core/embed/rust/src/ui/screens.rs

```diff
@@ -8,33 +8,26 @@ pub use super::model_tt::screens::*;
 use crate::ui::util::from_c_str;
 
 #[no_mangle]
-extern "C" fn screen_fatal_error_c(msg: *const cty::c_char, file: *const cty::c_char) {
-    let msg = if msg.is_null() {
-        None
-    } else {
-        unsafe { from_c_str(msg) }
-    };
-    let file = if file.is_null() {
+extern "C" fn screen_fatal_error_r(
```

Comment:
What is the meaning of `_r` suffix? If it is `Rust`, I would write `_rust` as my first thought was `model R`

Reply:
https://github.com/trezor/trezor-firmware/commit/fafa8e4bd46f5f04e2266ae88570675357b4a695

note that in `rust_ui.h` there is another similarly named function `loader_uncompress_r`, i left it like that for now as it will be hopefully removed entirely in near future.